backend/src/main/java/feedzupzup/backend/s3/controller/S3Controller.java

```diff
@@ -0,0 +1,24 @@
+package feedzupzup.backend.s3.controller;
+
+import feedzupzup.backend.global.response.SuccessResponse;
+import feedzupzup.backend.s3.api.S3Api;
+import feedzupzup.backend.s3.dto.request.PresignedUrlRequest;
+import feedzupzup.backend.s3.dto.response.PresignedUrlResponse;
+import feedzupzup.backend.s3.service.S3Constants;
+import feedzupzup.backend.s3.service.S3Service;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+public class S3Controller implements S3Api {
+
+    private final S3Service s3Service;
+
+    public SuccessResponse<PresignedUrlResponse> generatePresignedUrl(final PresignedUrlRequest request) {
+        final PresignedUrlResponse response = s3Service.requestPresignedUrl(
+                request.feedbackId(), request.extension(), S3Constants.FEEDBACK_OBJECT_DIR);
```

Comment:
p2)
현재 S3Service는 특정 도메인을 지정하지 않고, objectId라는 변수명을 사용하고 있는데,
dto명도 같이 objectId로 고쳐줘야할 것 같아요~

```suggestion
        final PresignedUrlResponse response = s3Service.requestPresignedUrl(
                request.objectId(), request.extension(), S3Constants.FEEDBACK_OBJECT_DIR);
```